### ISSUE TITLE
"bot.get_media_id_from_link" is written twice.

### ISSUE THREAD

**Issue by enganese** [instagrambot/docs#24]
![Screenshot_20211227-221849_Chrome](https://user-images.githubusercontent.com/69082498/147494494-5b9ce2b0-4415-4f26-b48a-c96239f27c67.jpg)

check the code example, there are two "bot.get_media_id_from_link". the second is supposed to be "bot.get_link_from_media_id"

**Comment by heetu** [comment]
@enganese yes there is some minor mistake in the documentation, I hope the repo moderator solves these issues.
